src/main/config/config_master.h

```diff
@@ -161,6 +161,9 @@ typedef struct master_t {
 
     uint8_t magic_ef;                       // magic number, should be 0xEF
     uint8_t chk;                            // XOR checksum
+    
+    char name[MAX_NAME_LENGTH];
```

Comment:
you should probably go with MAX_NAME_LENGTH+1 to allow for a null terminator (if you consider max_name_length. the usable length). That way any null terminated string handling routines can be used safely.